cg/meta/clean/demultiplexed_flow_cells.py

```diff
@@ -56,28 +56,30 @@ def __init__(
         self.identifier: str = self.split_name[FLOW_CELL_IDENTIFIER_POSITION]
         self.sequencer_serial_number: str = self.split_name[SEQUENCER_IDENTIFIER_POSITION]
         self.id: str = self.identifier[1:]
-        self._hk_fastq_files = None
-        self._hk_spring_files = None
-        self._is_correctly_named = None
-        self._exists_in_statusdb = None
-        self._fastq_files_exist_in_housekeeper = None
-        self._spring_files_exist_in_housekeeper = None
-        self._files_exist_in_housekeeper = None
-        self._files_exist_on_disk = None
-        self._passed_check = None
-        self._sequencer_type = None
-        self._is_demultiplexing_ongoing_or_started_and_not_completed = None
+        self._hk_fastq_files: Optional[list] = None
+        self._hk_spring_files: Optional[list] = None
+        self._is_correctly_named: Optional[bool] = None
+        self._exists_in_statusdb: Optional[bool] = None
+        self._fastq_files_exist_in_housekeeper: Optional[bool] = None
+        self._spring_files_exist_in_housekeeper: Optional[bool] = None
+        self._files_exist_in_housekeeper: Optional[bool] = None
+        self._files_exist_on_disk: Optional[bool] = None
+        self._passed_check: Optional[bool] = None
+        self._sequencer_type: Optional[str] = None
+        self._is_demultiplexing_ongoing_or_started_and_not_completed: Optional[bool] = None
 
     @property
     def sequencer_type(self) -> str:
-        """The type of sequencer a flow cell has been sequenced on"""
+        """The type of sequencer a flow cell has been sequenced on."""
         if self._sequencer_type is None:
-            self._sequencer_type = sequencer_types.get(self.sequencer_serial_number, "other")
+            self._sequencer_type: str = sequencer_types.get(
```

Comment:
I took a look at the sequencer_types dict and we're missing our latest Novaseq: [A01901](https://atlas.scilifelab.se/infrastructure/laboratory_equipment/sequencing_equipment/novaseq/#22-instruments-and-equipment). Not sure how that has gone unnoticed

Reply:
Added